libdyson/__init__.py

```diff
@@ -64,4 +65,6 @@ def get_device(serial: str, credential: str, device_type: str) -> Optional[Dyson
         return DysonPureHotCool(serial, credential, device_type)
     if device_type == DEVICE_TYPE_PURE_HUMIDIFY_COOL:
         return DysonPureHumidifyCool(serial, credential, device_type)
+    if device_type == DEVICE_TYPE_PURIFIER_HUMIDIFY_COOL_FORMALDEHYDE:
```

Comment:
It would be great to add tests for this part

Reply:
I'm sorry. Fixed.